### PR TITLE
Header and methodology added

### LINKED ISSUE #8: Finalize copy in the app
**What needs to be done? Why does it need to be done?**
Make sure all of the copy matches what lives in the [PRD](https://docs.google.com/document/d/1vssQ4JB8VyfInFpdw7F_4laQG0WVXYv6u4FwIpIFUoQ/edit?ts=5f58fbbc) and mocks.

This includes the title and body text in the blue section at the top, the methodology text at the bottom, all of the labels and elements in the central component, and the dynamic text updated by the various interactive elements. This may all have been done previously, but this task lives to ensure we double-check everything and confirm it reads and works as expected.

**Additional context**
N/A

### LINKED ISSUE #25: Add background elements and copy
**Is your feature request related to a problem? Please describe.**
The mocks in the PRD include a background behind the central component, with a dark blue background on the top with a headline and body text, and a light grey background on the bottom with a smaller headline and body text. We don't yet know exactly how this will be embedded in its final place, so let's go ahead and include those background elements in our package.

**Describe the solution you'd like**
Ensure that our component includes the background elements described above, as specified by the mocks. Ideally, this would be built in such a way that if it turns out we don't need to create those elements ourselves, they will be easy to remove.

**Describe alternatives you've considered**
N/A

**Additional context**
Final copy is to be provided as part of #8.

### PR DESCRIPTION
**Pull request by Wmaileq**
## Description of the change

- added header and methodology text examples to the app
- refactored css to scss (to have better developer experience while development of mobile mock up)

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Part of #8
Closes #25 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by jessex** [comment]
Note, @Wmaileq, that I changed you PR description to _not_ close #8, since the copy is still not finalized. For example, there is still "Lorem ipsum" text at the top.

Also, make sure to resolve the new merge conflicts when you get a chance.

File: src/SupervisionSuccess/SupervisionSuccess.js [hunk 1]
```diff
@@ -60,6 +63,14 @@ const SupervisionSuccess = ({ params }) => {
   );
 };
 
+SupervisionSuccess.defaultProps = {
+  title: "Supervision Success",
+  methodology: `Savings per day is calculated as a graduated function of the population differential. 
+  If the population differential surpasses any of several thresholds for closing facilities,
+  the total operating budget of thos facilities is saved. For remaining population differentials-namely
```
Comment (jessex) [inline src/SupervisionSuccess/SupervisionSuccess.js:70]: Typo: thos -> those

**Review by jessex — CHANGES_REQUESTED** [review]
Looking good so far, @Wmaileq! No comments on the code. Thanks for the switch to SCSS in the component styling.

One UI thing I notice is that the background color in the bottom section is currently white, but it's a light grey in the mocks. Reminder that Juan Agron added links to assets, fonts, colors, etc. in a comment on September 17th that is attached to the mocks in the PRD. We should make sure we're matching those--I think we're pretty good, and that lower background color is the only thing I've noticed that's off so far, but it's worth double-checking.

**Comment by Wmaileq** [comment]
Actually i don't think that the header text and the methodology text should be in-scope of the component.
Maybe I should take this components out of SupervisionSuccess component and reproduce them in App.js just to show an example how it could be realized on final page?

**Comment by jessex** [comment]
[Right here, Vlad](https://docs.google.com/document/d/1vssQ4JB8VyfInFpdw7F_4laQG0WVXYv6u4FwIpIFUoQ/edit?disco=AAAAG81Tf7o)

**Review by jessex — APPROVED** [review]
Looks good, @Wmaileq, thanks! I'm going to merge this PR. But take a look at that linked comment tomorrow and if you see anything that we're not yet matching, then let's file a follow-up PR to match it.

**Comment by Wmaileq** [comment]
> [Right here, Vlad](https://docs.google.com/document/d/1vssQ4JB8VyfInFpdw7F_4laQG0WVXYv6u4FwIpIFUoQ/edit?disco=AAAAG81Tf7o). Just follow whatever is linked in that comment.

Looks like i have no access to comments.

**Comment by jessex** [comment]
Sorry about that, Vlad! Just sent you an email. We can continue the conversation there.

**Pull request merged**